src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts

```diff
@@ -14,14 +16,17 @@ export class PresentationDefinitionWrapperEntity {
   _id: ObjectID;
 
   @Column()
+  @IsDefined({ message: 'PresentationDefinitionWrapper.presentation_definition must be provided' })
   // @ts-ignore
-  presentation_definition: IPresentationDefinition
+  presentation_definition: IPresentationDefinition;
 
   @Column()
+  @IsNotEmpty({ message: 'PresentationDefinitionWrapper.id is invalid' })
   // @ts-ignore
   id: string;
```

Comment:
PK

Reply:
That one needs to match the presentation id afaik, so I will leave it as it is